File: python/01-blink.py
import RPi.GPIO as GPIO
import time

# Run `gpio readall` to confirm BCM pin number(s)
pinOut = 17	# LED

GPIO.setmode(GPIO.BCM)
GPIO.setwarnings(False)
GPIO.setup(pinOut, GPIO.OUT)

def blink(pin):
  GPIO.output(pin,GPIO.HIGH)
  time.sleep(0.5)
  GPIO.output(pin,GPIO.LOW)
  time.sleep(0.5)
  return

try:
  for i in range(0,5):
    blink(pinOut)
finally:
  GPIO.cleanup()
  print("\nDone")

File: ruby/01-blink.rb
require 'rpi_gpio'

# Run `gpio readall` to confirm BCM pin number(s)
pin_out = 17	# LED

RPi::GPIO.set_numbering(:bcm)
RPi::GPIO.set_warnings(0)
RPi::GPIO.setup(pin_out, as: :output)

def blink(pin)
	RPi::GPIO.set_high(pin)
	sleep(0.5)
	RPi::GPIO.set_low(pin)
	sleep(0.5)
end

5.times { blink(pin_out) }

RPi::GPIO.clean_up